src/main/kotlin/racing/car/RacingCar.kt

```diff
@@ -0,0 +1,9 @@
+package racing.car
+
+class RacingCar {
+    var position = 0
```

Comment:
private으로 정의하지 않으셨기 때문에 외부에서 car.position = 100 요런식으로 변경이 가능하겠네요.

자바와 같이 무조건 privated으로 정의하고 getter, setter를 선언할 필요는 없으나 이런 경우라면 private set을 지정하셔서 외부에서의 수정을 막는게 좋아보입니다.

```suggestion
    var position = 0
        private set
```